.pipelines/templates/.builder-release-template.yaml

```diff
@@ -1,13 +1,55 @@
 parameters:
-    - name: artifactName
-      type: string
-      default: 1604
-
+  - name: artifactName
+    type: string
+    default: 1604
+  - name: useOverrides
+    type: boolean
+    default: false
+  - name: overrideBranch
+    type: string
+    default: master
 steps:
+  - checkout: self
+    # s is the default path for repositories - if we don't set this when using multiple repsositories, then it is the repo name
+    # "s/AgentBaker", but most of the pipeline variables still refer to "s" and many of the scripts below fail.
+    path: s
+    fetchTags: false
+    fetchDepth: 1
+  - checkout: git://CloudNativeCompute/aks-rp@${{ parameters.overrideBranch }}
+    path: aks-rp
+    fetchDepth: 1
+    fetchTags: false
+    lfs: false
+    condition: eq('${{ parameters.useOverrides }}', true)
+  - bash: |
+      set -euo pipefail
+      # Just copy components.json. If anything else is needed, then someone can modify the pipeline in an emergency.
+      # we check for existence of both src and destination file. As this pipeline will only be run if an override is happening,
+      # then we it's an error to not do an override.
+      COMPONENT_JSON_SRC=$(Pipeline.Workspace)/aks-rp/agentbakersvc/overrides/components.json
+      COMPONENT_JSON_DEST=./parts/linux/cloud-init/artifacts/components.json
+      if [ ! -f "${COMPONENT_JSON_DEST}" ]; then
+        echo destination components.json file does not exist, not copying as it must have been moved in a refactor: ${COMPONENT_JSON_DEST}
+        exit 1
+      else
+        echo Found file to overwrite: ${COMPONENT_JSON_DEST}
+      fi
+
+      if [ ! -f "${COMPONENT_JSON_SRC}" ]; then
+        echo src components.json file does not exist, not copying: ${COMPONENT_JSON_SRC}
```

Comment:
```suggestion
        echo "src components.json file does not exist, not copying: ${COMPONENT_JSON_SRC}"
```

Reply:
Github won't let me commit this, not sure why, so have done it myself.